api/src/main/java/org/ehcache/config/CacheConfiguration.java

```diff
@@ -99,5 +99,22 @@
    *
    * @return {@code true} if the {@code Cache} needs to be persistent, {@code false} otherwise
    */
-  boolean isPersistent();
+  PersistenceMode getPersistenceMode();
+
+  /**
+   * Get the {@link ResourcePools resource pools} the {@link Cache} can make use of.
+   *
+   * @return the {@link ResourcePools}
+   */
+  ResourcePools getResourcePools();
```

Comment:
If they're exclusive to this cache, should we just call them "Resources"?  And should we use a collection type here?